### ISSUE TITLE
[VT-90] Update display after sync

### ISSUE THREAD

**Issue by tbabej** [tbabej/vit#90]
_Jochen Sprickerhof on 2014-06-12T07:25:41Z says:_ 

 Currently task are not reread after syncing with taskd. The attached patch fixes this.

**Comment by tbabej** [comment]
Migrated metadata:
```
Created: 2014-06-12T07:25:41Z
Modified: 2015-03-01T23:43:00Z
```

**Comment by tbabej** [comment]
_Scott Kostyshak on 2015-03-01T00:59:25Z says:_ 

 Thank you for this bug report. I just took a quick look at your patch. From what I understand, you suggest a reread after ever character input. I don't think this is correct. I think we can fix this bug in a more simple way, but just rereading after syncing. I will look into this for 1.3.

**Comment by tbabej** [comment]
_Jochen Sprickerhof on 2015-03-01T17:09:47Z says:_ 

 Putting the reread_needed after the &shell_exec("task sync" should be fine as well. Thanks for coming back to this.

**Comment by tbabej** [comment]
_Scott Kostyshak on 2015-03-01T23:42:00Z says:_ 

 I actually misread the patch on my quick look. Yes, I do like putting it after the shell_exec. It reads better and is more efficient. I just committed to 117255b2 (on branch 1.3). Thank you for the report and patch. You're in the AUTHORS file. Note that I kept the refresh_needed that you removed. It is necessary (otherwise the error is not shown because the screen is not refreshed).
